app/src/androidTest/java/com/github/studydistractor/sdp/history/FakeHistoryModule.kt

```diff
@@ -20,12 +15,12 @@ import java.util.*
 )
 class FakeHistoryModule {
     @Provides
-    fun provideHistoryInterface() : HistoryInterface{
+    fun provideFakeHistoryInterface() : HistoryInterface {
         return FakeHistoryInterface()
     }
 }
 
-class FakeHistoryInterface : HistoryInterface {
+class FakeHistoryInterface @Inject constructor(): HistoryInterface {
```

Comment:
I strongly disagree with the name. It should be "History". [see this CodeAesthetic vid](https://www.youtube.com/watch?v=-J3wNP6u5YU)

Reply:
This is not one of my changes, but coming from the commit I wrote my code on top of.